server.js

```diff
@@ -92,6 +92,12 @@ app.use(function (req, res, next) {
   next();
 });
 
+// Set Cache-Control to no-store to stop states from being cached.
+app.use(function (req, res, next){
+  res.setHeader("Cache-Control", "no-cache,no-store,max-age=0");
```

Comment:
@rpowis Yeah I wondered about whether I should do that. At the time I decided to keep the distinct functionality separate. I can bundle them in together though if you'd prefer.